CommandFactory.py

```diff
@@ -0,0 +1,10 @@
+from CreateCommand import CreateCommand
+from ParseInput import parse_args
+args= parse_args()
+class CommandFactory:
+    @staticmethod
+    def build_command(args):
+        command_type=args
+        if command_type == "create":
+            return CreateCommand().excute()
```

Comment:
Factory  should create the command line. It shouldn't execute it. I think I mentioned this in the session. You are createing the command here but you execute it later in the main

Reply:
I don't understand this part. When I remove the ``` .excute() ```, the program doesn't do anything

Reply:
As I explained, u will call it from the main